pages/02.Documentation/02.compute/docs.en.md

```diff
@@ -0,0 +1,162 @@
+
+SysEleven Stack distributed storage distributes several copies of segments of your data over many physical ssd devices attached to different physical compute nodes connected via network. This allows for high overall performance, because several devices can work simultaneously, but introduces a latency for single operations, because data has to be transmitted via network.
+
+SysEleven Stack local ssd storage stores your data on a local raid mirrored ssd storage directly attached to the compute node. This reduces the latency, because no network is involved, but also redundancy, because only two devices and one compute node are involved.
+
+### Which storage flavor fits my needs best?
+
+In general, workloads where large volumes of data are transmitted or many small chunks of data are handled in parallel benefit from the overall performance of distributed storage and of course the redundancy and availability whereas workloads with tiny requests that need to be executed serially benefits from the lower latency of local ssd storage.
+
+### Why are instances migrated?
+
+#### Software Updates
+
+SysEleven regularly updates the software on the hypervisor host machines.
+To apply certain updates a reboot is required and running instances are therefore moved to another
+hypervisor host.
+
+#### Hardware Maintenance
+
+All hardware nodes require maintenance at some point. Sometimes the required maintenance work cannot be done
+while the machine is online. Therefore instances are moved to another hardware node prior to the planned maintenance work.
+
+#### Hardware failure
+
+Unfortunately life migrations are not possible in case of a hardware failure, therefor running instances will be automatically restarted on another hardware node. Stopped instances will be moved but remain in their stopped state.
+
+### How long does a migration take?
+
+A live migration takes usually about 500ms. In some situations migrations may take longer.
+
+### Why are instances disconnected while migrating?
+
+To transfer the active state of instances (incl. RAM/Memory) they need to be 'frozen' prior to the migration. During the transfer network packets can get lost. It depends on the operating system and application that is being used if connection can be reestablished.
+
+### Can I allocate a fixed IP to a compute instance?
+
+Normally a fixed IP shouldn't play a big role in a cloud setup, since the infrastructure might change a lot.
+If you need a fixed IP, you can assign a port from our networking service as a fixed IP to our compute instance. Here is an example which shows how to use the orchestration service to fetch a fixed IP address to use in a template:
+
+```plain
+  management_port:
+    type: OS::Neutron::Port
+    properties:
+      network_id: { get_resource: management_net }
+      fixed_ips:
+        - ip_address: 192.168.122.100
+```
+
+### My compute instance was created, but is not accessible via SSH/HTTP
```

Comment:
why is the "etc." gone?

Reply:
Syntax error. not allowed to use in headings